templates/managed-rds-postgresql-v1.j2

```diff
@@ -0,0 +1,284 @@
+  DBBackupRetentionPeriod:
+    Description: The number of days for which automated backups are retained. Allowed
+      values 0-10. Setting this parameter to a positive number enables backups. Setting
+      this parameter to 0 disables automated backups.
+    Default: '1'
+    Type: Number
+    MinValue: '0'
+    MaxValue: '10'
+    ConstraintDescription: must be a number between 0 and 10.
+  VolumeSize:
+    Default: '20'
+    Description: General Purpose (SSD) storage (gp2). Must be an integer from 20 to
+      16384.
+    Type: Number
+    MinValue: '20'
+    MaxValue: '16384'
+    ConstraintDescription: Must be an integer from 20 to 16384.
+  EncryptVolume:
+    Default: 'true'
+    Description: Local encryption on the instance
+    Type: Boolean
+  DBEngineVersion:
+    Description: The database engine version
+    Type: String
+    Default: '11.2'
+    AllowedValues:
+      - '11.5R1'
+      - '11.2'
+      - '11.1'
+      - '10.7'
+      - '10.6'
+      - '10.5'
+      - '10.4'
+      - '10.3'
+      - '10.1'
+  InstanceType:
+    Description: The database instance type
+    Type: String
+    Default: db.t2.small
+    AllowedValues:
+      - db.t2.micro
+      - db.t2.small
+      - db.t2.medium
+      - db.t2.large
+      - db.t2.xlarge
+      - db.t2.xlarge
+      - db.m5.24xlarge
+      - db.m5.12xlarge
+      - db.m5.4xlarge
+      - db.m5.2xlarge
+      - db.m5.xlarge
+      - db.m5.large
+      - db.m4.large
+      - db.m4.xlarge
+      - db.m4.2xlarge
+      - db.m4.4xlarge
+      - db.m4.10xlarge
+      - db.m4.16xlarge
+      - db.m3.medium
+      - db.m3.large
+      - db.m3.xlarge
+      - db.m3.2xlarge
+      - db.r3.large
+      - db.r3.xlarge
+      - db.r3.2xlarge
+      - db.r3.4xlarge
+      - db.r3.8xlarge
+      - db.r4.large
+      - db.r4.xlarge
+      - db.r4.2xlarge
+      - db.r4.4xlarge
+      - db.r4.8xlarge
+      - db.r4.16xlarge
+    ConstraintDescription: must select a valid database instance type.
+  MultiAZ:
+    Description: Multi-AZ database
+    Type: String
+    Default: 'true'
+    AllowedValues:
+      - 'true'
+      - 'false'
+    ConstraintDescription: must be true or false.
+Conditions:
+  PublicDBResource: !Or [!Equals [ !Ref VpcSubnet, PublicSubnet ], !Equals [ !Ref VpcSubnet, PublicSubnet1 ], !Equals [ !Ref VpcSubnet, PublicSubnet2 ] ]
+  PrivateDBResource: !Or [!Equals [ !Ref VpcSubnet, PrivateSubnet ], !Equals [ !Ref VpcSubnet, PrivateSubnet1 ], !Equals [ !Ref VpcSubnet, PrivateSubnet2 ] ]
+  HasMksKeyId: !Not [!Equals ["", !Ref KmsKeyId]]
+Resources:
+  {% if sceptre_user_data.IngressWhitelist is defined %}
+  DBInstanceSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: "Open database on defined port for access by whitelisted IP"
+      VpcId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-${VpcName}-VPCId'
+      SecurityGroupIngress:
+      {% for block in sceptre_user_data.IngressWhitelist %}
```

Comment:
[optional] would be more clear if `block` is `ip` or `CidrIp`

Reply:
IO'll use CIDRBlock to be specific